files/www/cgi-bin/status

```diff
@@ -100,6 +100,21 @@ function get_default_gw()
     return "none"
 end
 
+function get_memavail()
+    local f = io.open("/proc/meminfo", "r")
+    if f then
+        for line in f:lines()
+        do
+            local memavail = line:match("^MemAvailable:%s+(%d+)%s+kB")
+            if memavail then
+                f:close()
+                return memavail
+            end
+        end
+    end
+    f:close()
```

Comment:
I'm sure it was my comment placement, but the f:close should be inside the if ... end not after it.

Reply:
Doh.   Thanks again Tim